Gemfile.lock

```diff
@@ -2,7 +2,7 @@ GEM
   remote: https://rubygems.org/
   specs:
     Ascii85 (1.1.0)
-    MailchimpMarketing (3.0.78)
+    MailchimpMarketing (3.0.80)
```

Comment:
Same note here – we don't need to update all the gems in this PR to update the the buttons. Let's leave for a separate PR.